app/views/resorts/index.html.erb

```diff
@@ -1,5 +1,13 @@
 <h1>All Resorts</h1>
 
+<%= link_to 'Add New Resort', "/resorts/new" %>
+
 <% @resorts.each do |resort| %>
-  <h3><%= resort %>Created at: <%= resort.created_at%></h3>
+  <p>Name: <%= resort.name %></p>
```

Comment:
Yeah, I think this is a better way to display all the resorts. I'm going to add this to my list to refactor, the Instructor index looks pretty sad without all the data.